### PR TITLE
Update spielman email and affil

### PR DESCRIPTION
**Pull request by sjspielman**
### Purpose

Stephanie now works at CCDL and author credentials need to be updated accordingly.

### Issue

_What GitHub issue does your pull request address?_
#179 

#### Which areas should receive a particularly close look?

I updated email and affiliation, but I'm not sure how to add a superscript in this format to indicate _current_ affiliation.

#### Is there anything that you want to discuss further?

Superscript!

#### Is the pull request ready for review?

yes

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [ ] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### Spellcheck Step

The dictionary used for spellchecking can be updated.
Edit the file in `build/assets/custom-dictionary.txt` by adding new entries to the end.
You do not need to change anything else.
However, if you want to update the first line to have an accurate count of words and you want to remove non-unique ones, run the following command from within `build/assets` on your favorite OS X or Linux machine:
```
(( len = $(awk '!a[$0]++' < custom-dictionary.txt | wc -l ) - 1 )); tmpfile="$(mktemp)"; echo "personal_ws-1.1 en $len utf-8" > $tmpfile; tail -n +2 custom-dictionary.txt | awk '!a[$0]++' >> $tmpfile; mv $tmpfile custom-dictionary.txt
```

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -169,14 +169,15 @@ authors:
...
     twitter: stephspiel
-    email: spielman@rowan.edu
+    email: stephanie.spielman@ccdatalab.org
     contributions:
       - Validation
       - Formal analysis
       - Writing - Review and editing
       - Visualization
       - Supervision
     affiliations:
```
[the remark this answers is not in the document]
Comment (jaclyn-taroni) [reply content/metadata.yaml:179]: Can you mention me once you've given this a try (or if you need help before that, of course)? 😄

[the remark this answers is not in the document]
Comment (sjspielman) [reply content/metadata.yaml:179]: I've added those and now waiting to see what the build looks like so I can modify accordingly so the symbol works as current affil.

**Comment by sjspielman** [comment]
@jaclyn-taroni I need help for sure. The symbol is nicely there on its own line, but getting is _as_ a superscript remains unclear to me.

File: content/00.front-matter.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ on {{manubot.date}}.
 {## Template for listing authors ##}
 {% for author in manubot.authors %}
 + **{{author.name}}**<br>
+  <sup>{%- if author.symbol_str is defined -%}{{author.symbol_str}}{%- endif -%}</sup><br>
```
Comment (jaclyn-taroni) [inline content/00.front-matter.md:31]: Fairly sure the break is the problem
```suggestion
+ **{{author.name}}**<sup>{%- if author.symbol_str is defined -%}{{author.symbol_str}}{%- endif -%}</sup><br>
```

Comment (jaclyn-taroni) [reply content/00.front-matter.md:31]: Going to commit to see...

**Comment by jaclyn-taroni** [comment]
I also expect something like

```
<sup>☯</sup>: Current affiliation Childhood Cancer Data Lab ...
```

in the front matter

**Comment by sjspielman** [comment]
> I also expect something like in the front matter

Ah! Yes, of course.

**Comment by sjspielman** [comment]
I like how the most recent version looks. Let me know what you think when this build finishes @jaclyn-taroni !

**Review by jaclyn-taroni — APPROVED** [review]

**Pull request merged**